### ISSUE TITLE
Latest release? 0.5

### ISSUE THREAD

**Issue by tommyms** [YunoHost-Apps/kimai2_ynh#6]
Just wondering if the latest release is coming soon? Version 0.5

**Comment by kevinpapst** [comment]
Thanks for posting @tommyms - you were quicker than me.

@anmol26s - regarding the upgrade, I'd Kindly ask you to change the version detection for the future from https://github.com/YunoHost-Apps/kimai2_ynh/blob/master/scripts/upgrade#L80 to something like:
```bash
bin/console kimai:version --short 2> /dev/null || sed -n 's/^future-release=//p' .github_changelog_generator
```
I guarantee that the version command `bin/console kimai:version --short` will always be there, but I am not sure about the `.github_changelog_generator` file. It won't go away soon, but its a part of the software management cycle and not part of the software directly.

**Comment by anmol26s** [comment]
@tommyms 
Thanks for the request
@kevinpapst 
Sure will use this from next release.

How can I shift make a shift from sqlight to mysql? I will be testing msql for version 0.5.

**Comment by anmol26s** [comment]
@tommyms 
The testing branch is ready with the update 0.5. You can try it if you want to update 0.5 now. 
You can update by following command:

`$ yunohost app upgrade -u https://github.com/YunoHost-Apps/kimai2_ynh/tree/testing kimai2 --debug`

**Comment by anmol26s** [comment]
@tommyms 
I have made a package which use the mysql package. So you can try that one too,but it will not do anything special. I think mysql would be better. But I don't know how to migrate from sqlite. Maybe @kevinpapst can help to migrate.

`$ yunohost app upgrade -u https://github.com/YunoHost-Apps/kimai2_ynh/tree/mysql kimai2 --debug`

**Comment by tommyms** [comment]
@anmol26s thanks! We have some logged data already that we would need to migrate over before changing to MySQL somehow.

**Comment by anmol26s** [comment]
Migration from sqlite to mysql would require you to do manual steps and it would not be easy. As there is no import and export feature implemented in the web interface yet. I would suggest you to keep using sqlite till import and export feature is implemented.
@kevinpapst would be able to suggest you migrate process in more detail and maybe its possible to migrate from command-line.

**Comment by anmol26s** [comment]
Can this be closed as the app has been updated to version 0.5?

**Comment by tommyms** [comment]
Yes @anmol26s I think it's all good now. Upgraded without issue.
